src/main/resources/lang/french.lang

```diff
@@ -0,0 +1,190 @@
+# Default French language file
+
+# Which version of Skript this language file was written for
+version: @version@
+
+# What null (nothing) should show up as in a string/text
+none: <none>
+
+# -- Skript --
+skript:
+	copyright: ~ créé par & © Peter Güttinger alias Njol ~
+	prefix: <gray>[<gold>Skript<gray>] # not used
+	quotes error: Utilisation incorrecte des guillemets ("). Si vous voulez utiliser des guillemets dans du texte "entre guillemets", doublez-les : "".
+	brackets error: Quantité ou emplacement des accolades incorrect. Assurez-vous que chaque accolade ouvrante corresponde à une accolade fermante.
+	invalid reload: Skript ne peut être rechargé qu'avec les commandes '/reload' de Bukkit ou '/skript reload' de Skript.
```

Comment:
Wouldn't it be more correct to say `avec la commande 'X' de Bukkit ou 'Y' de Skript"` ?